### PR TITLE
Add support for Google Chromebooks

### PR DESCRIPTION
**Pull request by archisman-panigrahi**
Some ARM Chromebooks support running Debian and the Pixel desktop of RPi OS.

This commit adds support for them

### CODE REVIEW

**Comment by archisman-panigrahi** [comment]
Please release a new version after merging, so that it would reach the users.

File: rpi_backlight/__init__.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ class BoardType(Enum):
     #: Tinker Board 2
     TINKER_BOARD_2 = 3
     #: Microsoft Surface RT
-    MICROSOFT_SURFACE_RT = 4
+    SURFACE_RT_OR_GOOGLE_CHROMEBOOK = 4
```
Comment (linusg) [inline rpi_backlight/__init__.py:27]: Please call it GENERIC at this point

File: rpi_backlight/cli.py [hunk 2]
```diff
@@ -6,14 +6,14 @@
...
-    "microsoft-surface-rt": BoardType.MICROSOFT_SURFACE_RT,
+    "microsoft-surface-rt": BoardType.SURFACE_RT_OR_GOOGLE_CHROMEBOOK,
 }
 
 BOARD_TYPE_TO_STRING = {
     BoardType.RASPBERRY_PI: "raspberry-pi",
     BoardType.TINKER_BOARD: "tinker-board",
     BoardType.TINKER_BOARD_2: "tinker-board-2",
-    BoardType.MICROSOFT_SURFACE_RT: "microsoft-surface-rt",
+    BoardType.SURFACE_RT_OR_GOOGLE_CHROMEBOOK: "microsoft-surface-rt",
```
Comment (linusg) [inline rpi_backlight/cli.py:16]: This needs updating too

File: rpi_backlight/__init__.py (hunk 1 above)
Comment (archisman-panigrahi) [reply rpi_backlight/__init__.py:27]: The problem with generic is that, most generic devices have brightness control at `sys/class/backlight/acpi_video0` or `sys/class/backlight/intel_backlight` (and those devices would be supported by i386 version of Raspberry Pi OS). Given that, what do you think we should do?

Comment (linusg) [reply rpi_backlight/__init__.py:27]: This is a library intended to run on a raspberry pi board with the official raspberry pi display, edge cases like this don't matter

**Comment by archisman-panigrahi** [comment]
How about now?

**Comment by linusg** [comment]
Looks good, thanks!

**Pull request merged**